conformance/tests/admin-network-policy-core-egress-tcp-rules.go

```diff
@@ -207,12 +208,65 @@ var AdminNetworkPolicyEgressTCP = suite.ConformanceTest{
 			// ensure egress from gryffindor is PASSED to slytherin at port 80; egressRule at index3 should take effect
 			success := kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-gryffindor", "harry-potter-0", "tcp",
 				serverPod.Status.PodIP, int32(80), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 			// harry-potter-1 is our client pod in gryffindor namespace
 			// ensure egress from gryffindor is ALLOWED to slytherin for rest of the traffic; matches no rules hence allowed
 			success = kubernetes.PokeServer(t, s.ClientSet, &s.KubeConfig, "network-policy-conformance-gryffindor", "harry-potter-1", "tcp",
 				serverPod.Status.PodIP, int32(8080), s.TimeoutConfig.RequestTimeout, true)
-			assert.Equal(t, true, success)
+			assert.True(t, success)
 		})
 	},
 }
+
+var AdminNetworkPolicyEgressNamedPort = suite.ConformanceTest{
+	ShortName:   "AdminNetworkPolicyEgressNamedPort",
+	Description: "Tests support for egress traffic on a named port using admin network policy API based on a server and client model",
+	Features: []suite.SupportedFeature{
+		suite.SupportAdminNetworkPolicy,
```

Comment:
hmm shall we keep this for now under `suite.SupportAdminNetworkPolicyNamedPorts` alone? Putting it also as `suite.SupportAdminNetworkPolicy` makes it a core rather than an extended feature or should we make this a core feature?